### ISSUE TITLE
Problem using multiple monitors.

### ISSUE THREAD

**Issue by x421** [ch11ng/exwm#325]
Hi, 

I am trying to get exwm working with my monitor setup. 

It is the following:
```
                               +----+
                 +-----------+ | M  |
+--------------+ |   Mon 1   | | O  |
|    Laptop    | +-----------+ | N  |
+--------------+               | 2  |
                               +----+
```

The first problem after I'm starting exwm is that emacs spans over all the monitors.
This problem is fixed by running (exwm-randr--refresh). But then I only get workspaces on MON2 (DP-3-2), the other monitors (Laptop/eDP-1 and Mon1/DP-3-1) are blank, displaying a black background.

My workspace list looks like this: 
```
'(0 "eDP-1" 1 "eDP-1" 2 "DP-3-1" 3 "DP-3-1" 4 "DP-3-2" 5 "DP-3-2")
```

What can I do to make exwm use all workspaces as startup?

Here is my xrandr config:
```
xrandr --output eDP-1 --mode 2560x1440 --pos 0x0 --rotate normal --output HDMI-3 --off --output HDMI-2 --off --output HDMI-1 --off --output DP-3-1 --primary --mode 1920x1200 --pos 2560x0 --rotate normal --output DP-3-3 --off --output DP-3-2 --mode 1920x1200 --pos 4480x0 --rotate left --output DP-3 --off --output DP-2 --off --output DP-1 --off
```

**EDIT 1**

I just tried to do a `(exwm-randr--refresh)` right after `(exwm-enable)` and got the following backtrace:
https://gist.github.com/x421/afd0472c0ef0df226c11505d7ab7e475

**EDIT 2**
The i3 wm handles the same xrandr configuration without a problem, so I guess I am doing something wrong in my exwm config. 

**EDIT 3** 
Tried it using a plain new config. Got the same error. Anyone who can point me in the right direction?

**Comment by ch11ng** [comment]
When did you configure with `xrandr`? You should never run `exwm-randr--refresh` manually.  You only need to configure the multi-monitor settings with `xrandr`.

Please show us the output of `xrandr` and the full configuration related to `exwm-randr`.

**Comment by x421** [comment]
I'm so sorry. I wanted to create a clean exwm config to reproduce the issue, when I found what was the problem with the last "clean" config: I called (exwm-config-default) before (exwm-randr-enable). With a new clean file everything is working. I guess the intial setup had some quirks outside of exwm because it's the first time I am using such a setup with linux (or at all). 

Thank you for your time and your effort. I am really glad you are doing exwm. I'm closing this.
